docs/resources/cluster_v2.md

```diff
@@ -563,6 +566,27 @@ The following attributes are exported:
 * `region` - (Optional) ETCD snapshot S3 region (string)
 * `skip_ssl_verify` - (Optional) Disable ETCD skip ssl verify. Default: `false` (bool)
 
+##### `rotate_certificates`
+
+###### Arguments
+
+* `generation` - (Optional) Desired certificate rotation generation (int)
```

Comment:
So what @jakefhyde and I concluded on was that the top level rke_config fields (cert rotation, snapshot create, snapshot restore) should be Optional but the proper subfields should be Required because for ex: you don't want to configure a restore without the name and generation. So if the generation has to be set for cert rotation to work, then I think it should be Required. @paynejacob please chime in

Reply:
I think this makes sense and we can require it.